### PR TITLE
Add document for node conformance test.

### PR DESCRIPTION
**Pull request by Random-Liu**
This PR added a document for the node conformance test to `Creating a Cluster/Running Kubernetes on Custom Solutions/Validate Node Setup`

@dchen1107 
/cc @kubernetes/sig-node

---
This change"s [Reviewable](https://reviewable.io/reviews/kubernetes/kubernetes.github.io/1216)

### CODE REVIEW

**Comment by davidopp** [comment]
Does this mean the node e2e tests have to be 100% non-flaky? Or will node conformance test retry a few times on failure?

**Comment by Random-Liu** [comment]
@davidopp By default, the node conformance test only runs the node e2e test with "[conformance]" label, which should be stable (or else we should fix it or remove the conformance tag)

However, add flake retry is a good idea for better user experience.

**Comment by devin-donnelly** [comment]
Docs LGTM. Needs a Tech LGTM.

**Comment by Random-Liu** [comment]
@dchen1107 PTAL.

**Comment by dchen1107** [comment]
LGTM

File: docs/admin/node-conformance.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+conformance test. They'll be improved in future version.
+
+* Node conformance test only supports Docker as the container runtime.
+* Node conformance test doesn't validate network related system configurations
+  and functionalities.
+
+## Prerequisite
+
+Node conformance test is used to test whether a node is ready to join a
+Kubernetes cluster, so the prerequisite is the same with a standard Kubernetes
```
Comment (yujuhong) [inline docs/admin/node-conformance.md]: Are there any documentation on the "prerequisite" of a standard kubernetes npde?

Comment (Random-Liu) [inline docs/admin/node-conformance.md]: I want to link somewhere, but I don't think we have it. :(

File: docs/admin/node-conformance.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+---
+
+* TOC
+{:toc}
+
+## Node Conformance Test
+
+*Node conformance test* is a test framework validating whether a node meets the
+minimum requirement of Kubernetes with a set of system verification and
+functionality test. A node which passes the test is qualified to join a
```
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: ACK

File: docs/admin/node-conformance.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  arm   |    node-test-arm  |
+ arm64  |  node-test-arm64  |
+
+### Run Selected Test
+
+In fact, Node conformance test is a containerized version of [node e2e
+test](https://github.com/kubernetes/kubernetes/blob/release-1.4/docs/devel/e2e-node-tests.md).
+By default, it runs all conformance test.
+
+Theoritically, you can run any node e2e test if you configure the container and
```
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: ACK.

File: docs/admin/node-conformance.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+# $LOG_DIR is the test output path.
+sudo docker run -it --rm --privileged --net=host \
+  -v /:/rootfs:ro -v /var/run:/var/run \
+  -v $CONFIG_DIR:/etc/manifest -v $LOG_DIR:/var/result \
+  gcr.io/google_containers/node-test-amd64:v0.1
+```
+
+### Run Node Conformance Test for Other Architectures
+
+We also build node conformance test docker image for other architectures:
```
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: ACK.

File: docs/admin/node-conformance.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+ arm64  |  node-test-arm64  |
+
+### Run Selected Test
+
+In fact, Node conformance test is a containerized version of [node e2e
+test](https://github.com/kubernetes/kubernetes/blob/release-1.4/docs/devel/e2e-node-tests.md).
+By default, it runs all conformance test.
+
+Theoritically, you can run any node e2e test if you configure the container and
+mount required volumes properly. But **it is strongly recommanded to only run conformance
```
Comment (Random-Liu) [inline docs/admin/node-conformance.md]: ACK

**Pull request merged**